### PR TITLE
update information for Typetalk action

### PR DESCRIPTION
**Pull request by dataich**
update description option's help and author.

### CODE REVIEW

File: lib/fastlane/actions/typetalk.rb [hunk 1]
```diff
@@ -52,21 +52,21 @@ def self.check_response(response)
       end
 
       def self.description
-        "Post a message to Typetalk"
+        "Post a message to Typetalk. http://www.typetalk.in/"
```
Comment (KrauseFx) [inline lib/fastlane/actions/typetalk.rb]: Sorry, no URLs in the description.

File: lib/fastlane/actions/typetalk.rb [hunk 2]
```diff
@@ -52,21 +52,21 @@ def self.check_response(response)
...
       end
 
       def self.available_options
         [
           ['message', 'The message to post'],
-          ['note_path', 'Path to a md file'],
-          ['topicId', ''],
+          ['note_path', 'Path to a additional note'],
```
Comment (KrauseFx) [inline lib/fastlane/actions/typetalk.rb]: Path to a**n** additional note

**Comment by KrauseFx** [comment]
Thanks @dataich, looks great :+1: Will be in the next release.

**Comment by dataich** [comment]
thanks😀

**Pull request merged**